gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -1107,13 +1190,41 @@ static gboolean
 gst_tensor_if_check_condition (GstTensorIf * tensor_if, GstBuffer * buf,
     gboolean * result)
 {
-  tensor_if_data_s cv = {.type = _NNS_END,.data._uint8_t = 0 };
+  if (tensor_if->cv == TIFCV_CUSTOM) {
+    GstMemory *in_mem[NNS_TENSOR_SIZE_LIMIT];
+    GstMapInfo in_info[NNS_TENSOR_SIZE_LIMIT];
+    GstTensorMemory in_tensors[NNS_TENSOR_SIZE_LIMIT];
+    guint i, j;
+
+    if (tensor_if->custom.func == NULL) {
+      const custom_cb_s *ptr =
+          get_subplugin (NNS_IF_CUSTOM, tensor_if->custom_name);
```

Comment:
get_subplugin is not that cheap. you may need to cache it (reload if set_property is called or related events occur)

Reply:
Changed to set custom properties in set_property.
And if a user tries to set a new custom option, simply overwrite it.